lib/datapacksutils.js

```diff
@@ -2162,4 +2163,13 @@ DataPacksUtils.prototype.deleteBulkRecords = async function (query, objectName,
             VlocityUtils.log('error: ' + err);
         });
     return promise;
-}
\ No newline at end of file
+}
+
+DataPacksUtils.prototype.loadIgnoreFile = function (projectPath) {
+    const ignoreFilePath = path.resolve(projectPath, '.vlocityignore');
+
+    if (fs.existsSync(ignoreFilePath)) {
+        console.log('ignorefile loaded');
```

Comment:
Don't use console.log statements

Reply:
forgot to remove it. it was just for testing purpose